--- pyopenfst-eb0c0f9af57bb597f9511e1cc69cbbefe50ff507/openfst_symtab.i.orig	2014-10-28 21:34:59.000000000 +0100
+++ pyopenfst-eb0c0f9af57bb597f9511e1cc69cbbefe50ff507/openfst_symtab.i	2018-03-12 19:10:25.371843803 +0100
@@ -19,7 +19,7 @@
     SymbolTable *Copy() const;
     %feature("docstring", "Read entries from a text file.");
     static SymbolTable* ReadText(std::string const & filename,
-                                 bool allow_negative = false);
+                                 const SymbolTableTextOptions &opts = SymbolTableTextOptions());
     %feature("docstring", "Write entries to a text file.");
     bool WriteText(std::string const & filename) const;
     %feature("docstring", "Read entries from a binary file.");
--- pyopenfst-eb0c0f9af57bb597f9511e1cc69cbbefe50ff507/openfst_properties.i.orig	2014-10-28 21:34:59.000000000 +0100
+++ pyopenfst-eb0c0f9af57bb597f9511e1cc69cbbefe50ff507/openfst_properties.i	2018-03-12 19:02:13.431849424 +0100
@@ -64,7 +64,7 @@
 uint64 ComposeProperties(uint64 inprops1, uint64 inprops2);
 uint64 ConcatProperties(uint64 inprops1, uint64 inprops2,
                         bool delayed = false);
-uint64 DeterminizeProperties(uint64 inprops, bool has_subsequential_label);
+uint64 DeterminizeProperties(uint64 inprops, bool has_subsequential_label, bool distinct_psubsequential_labels);
 //uint64 DifferenceProperties(uint64 inprops1, uint64 inprops2);
 uint64 FactorWeightProperties(uint64 inprops);
 //uint64 IntersectProperties(uint64 inprops1, uint64 inprops2);
@@ -75,7 +75,7 @@
 //                          ssize_t root,
 //                          bool epsilon_on_replace,
 //                          bool no_empty_fst);
-uint64 ReverseProperties(uint64 inprops);
+uint64 ReverseProperties(uint64 inprops, bool has_superinitial);
 uint64 ReweightProperties(uint64 inprops);
 uint64 RmEpsilonProperties(uint64 inprops, bool delayed = false);
 uint64 SynchronizeProperties(uint64 inprops);
